### DISCUSSION TITLE
lab1_timing hazards

### DISCUSSION THREAD

**Discussion by Stupid-WOLF** [Stupid-WOLF/EE332_SUSTech#2] [Q&A]
I have a question about the delay, how can we emiminate ~competitive adventure phenomenon of signls~ timing hazards.

**Comment by Stupid-WOLF** [comment]
so can anybody help me?

**Comment by fishress8421** [comment]
First of all, in Chinese, this phenomenon is called "冒险与竞争“, but in English they are usually called as timing hazards.

**Reply by Stupid-WOLF** [reply]
Thanks! I have modified it.
